docs/find-lost-trn-integration.md

```diff
@@ -77,18 +78,23 @@ redirect_uri=https%3A%2F%2Fauthserveruri%2F&client_title=The%20Client%20Title&em
 
 ## Handover from Find a lost TRN to the authorization server
 
-Once Find a lost TRN has completed its journey (whether successfully resolving a TRN or not), it needs to redirect back to the authorization server with the user's information.
-Find a lost TRN should POST to the `redirect_uri` specified in the initial request using the `application/x-www-form-urlencoded` content type.
-The request should include a `user` parameter. The `user` parameter must contain a JWT that includes the user's names, DOB, NINO and TRN, if known. The JWT should be signed with the same pre-shared key referenced above.
+Once Find a lost TRN has completed its journey (whether successfully resolving a TRN or not), it needs to provide the user information to the authorization server and redirect back there.
 
-When the authorization server receives the callback, it should validate the JWT has been signed by the pre-shared key. If validation fails an error should be shown; the authorization process must *not* complete.
+Find a lost TRN must do a back-end API call to a PUT endpoint on the authorization server's API `/api/find-trn/user/{journeyId}` (where `{journeyId}` is the ID passed in the initial handover request).
+The body is a JSON object with a single property - `subject`. This should contain a JWT that includes the user's names, DOB, NINO and TRN, if known. The JWT does not need to be signed.
 
-### Example:
+Finally, Find a lost TRN should redirect the user to the `redirect_uri` specified in the initial handover request.
+
+When the authorization server receives the callback, it will look up the user information that was persisted against the journey ID by the API call. Using that information it will register the user and completed the authorization process.
```

Comment:
```suggestion
When the authorization server receives the callback, it will look up the user information that was persisted against the journey ID by the API call. Using that information it will register the user and complete the authorization process.
```